Tools/ardupilotwaf/boards.py

```diff
@@ -201,6 +216,10 @@ def configure_env(self, cfg, env):
         env.LIB += [
             'm',
         ]
+        
+        env.CXXFLAGS += [
+            '-fexceptions',
```

Comment:
But SITL doesn't have a CAN implementation... Please remove this now and we'll deal with it when SITL is implemented.